scripts/templates/values.yaml

```diff
@@ -74,6 +79,11 @@ transformation: true
 #filtering: "filter('primary_aggregation_type', 'true')"
 filtering:
 
+## @param filtering_suffix - string - optional - default: ' and '
+## Determine the string to append at the end of the "filtering" param.
+#
+filtering_suffix: ' and '
```

Comment:
while you set a default in the template could you remove this value from this yaml?

Reply:
also if you can handle the spaces inside the template to not need to put them here when set this would be more user friendly

Reply:
> while you set a default in the template could you remove this value from this yaml?
done

> also if you can handle the spaces inside the template to not need to put them here when set this would be more user friendly
I prefer not to, leaving the space in the variable make the template more configurable.

If I move the space to the template I think we should rename the variable (as it's no longer a suffix).
If you still think we should move the space from the variable to the template, what variable name should we use ?

Reply:
I seen spaces as a safety feature which does not have limitation undesired impact but I can be wrong.

I am not against to force the user to put the spaces I just don't see the advantage.

Can you give me an example where it is useful ?